actions/build-push-to-dockerhub/action.yaml

```diff
@@ -48,10 +52,11 @@ runs:
         tags: ${{ inputs.tags }}
 
     - name: Build and push Docker image
-      uses: docker/build-push-action@3b5e8027fcad23fda98b2e3ac259d8d67585f671
+      uses: docker/build-push-action@v5.0.0
```

Comment:
@ricky-undeadcoders I think based on [this](https://github.com/grafana/shared-workflows/pull/43) we need to keep the reference here and not the semver

Reply:
Done, thanks for linking that.